Makefile.d/proto.mk

```diff
@@ -66,11 +67,17 @@ $(GOPATH)/src/github.com/protocolbuffers/protobuf:
 		https://github.com/protocolbuffers/protobuf \
 		$(GOPATH)/src/github.com/protocolbuffers/protobuf
 
-$(GOPATH)/src/github.com/googleapis/googleapis:
+$(GOPATH)/src/github.com/gogo/googleapis:
+	git clone \
+		--depth 1 \
+		https://github.com/gogo/googleapis \
+		$(GOPATH)/src/github.com/gogo/googleapis
+
+$(GOPATH)/src/github.com/gogo/protobuf:
 	git clone \
 		--depth 1 \
-		https://github.com/googleapis/googleapis \
-		$(GOPATH)/src/github.com/googleapis/googleapis
+		https://github.com/gogo/protobuf \
+		$(GOPATH)/src/github.com/gogo/protobuf
```

Comment:
Please tell me why the gogo/protobuf is used here.
Maybe as you know, gogo/protobuf is going to not to be maintain. (https://github.com/gogo/protobuf/issues/691)
So I think it is not a good choice to use it here. Is there any reasons we should use this?

Reply:
Because Vald currently uses gogo proto for serialization, it is not possible to serialize Status Details when using the standard proto.
Message. As a temporary measure, I replaced it with gogo proto.
Message. Although I know that it will no longer be maintained, go-protobuf v2 is still in beta, so I decided to continue using gogo proto, which still has the performance advantage considering that the only way to replace it is with the old std protobuf.
As for the replacement, since there are many changes in the I/F about the use of go standard protobuf v2, we need to consider it carefully and deal with it later.

Translated with www.DeepL.com/Translator (free version)

Reply:
ok, i see.